hangman/lib/hangman/application.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Hangman.Application do
+    use Application
+
+    def start(_type, args) do
+        import Supervisor.Spec, warn: false
+        
+        children = [
+            worker(Hangman, args)
+        ]
+
+        opts = [strategy: :one_for_one, name: Hangman.Supervisor]
```

Comment:
# -3

Your description was good, but the implementation here uses use_for_one.